src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalAmount = 0;
+
+  if (days < 3) {
+    totalAmount = days * 40;
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  if (days < 7) {
```
ти вже в попредньому if перевірила що воно більше 3, ще раз цього робити не треба

Reply:
updated